account/manager.go

```diff
@@ -766,12 +855,22 @@ func (m *manager) resumeAccount(ctx context.Context, account *Account, // nolint
 	// transaction to confirm so that we can transition the account to its
 	// final state.
 	case StatePendingClosed:
-		acctKey := account.TraderKey.PubKey.SerializeCompressed()
-		contextLabel := fmt.Sprintf(" poold -- "+
-			"AccountClosure(acct_key=%x)", acctKey)
+		txFee, err := m.deriveFeeFromTx(ctx, account.LatestTx)
+		fee := &txFee
+		if err != nil {
+			log.Errorf(
+				"Failed to derive fee from "+
+					"transaction: %v", account.LatestTx,
```

Comment:
nit: also log the causing error in these cases. So `... from transaction %v: %v, account.LatestTx, err)`).